ui/site/src/forum.ts

```diff
@@ -33,7 +33,9 @@ lichess.load.then(() => {
     $(el).replaceWith($('.forum-post__message', el));
   });
   $('.forum-post__message').each(function (this: HTMLElement) {
-    if (this.innerText.match(/(^|\n)>/)) {
+    const isVisible = window.getComputedStyle(this).display !== 'none';
+    const isMatching = isVisible ? this.innerText.match(/(^|\n)>/) : this.innerHTML.match(/(^|<br>)&gt;/);
```

Comment:
```js
this.innerText.match(isVisible ? /(^|\n)>/ : /(^|<br>)&gt;/);
```
still looks mysterious and fragile

Reply:
would this work since i check the <br>s with innerHtml 🤔

Reply:
no, sorry, I missed that distinction! my bad